src/lib/trips.js

```diff
@@ -135,3 +141,13 @@ const averageSpeedKmH = speeds => {
   // The speed is given in m/s
   return avgSpeed * 3.6
 }
+
+export const formatCalories = trip => {
+  const caloriesTrip = computeCaloriesTrip(trip)
+  return `${Math.round(caloriesTrip)} Kcal`
```

Comment:
We should neither use an uppercase. Neither in english nor in French. 

https://fr.wikipedia.org/wiki/Kilogramme => `Le kilogramme, dont le symbole est kg (en minuscules), est l'unité de base de masse dans le Système international d'unités (SI)2.`

https://en.wikipedia.org/wiki/Kilogram => `The kilogram (also kilogramme[1]) is the base unit of mass in the International System of Units (SI), the metric system, having the unit symbol kg.`


Same for kcal and so one